src/6_branch.js

```diff
@@ -1782,6 +1782,19 @@ if (WEB_BUILD) {
 			icon: options['icon'] || '',
 			title: options['title'] || '',
 			description: options['description'] || '',
+			reviewCount: (
+				typeof options['reviewCount'] === 'number' &&
+				options['reviewCount'] > 0 // force greater than 0
+			) ?
+				Math.floor(options['reviewCount']) : // force no decimal
+				null,
```

Comment:
Minor nit, but this could go on the same line:
) ? Math.floor(options['reviewCount']) : null